README.md

```diff
@@ -213,3 +214,143 @@ If you'd like to do this process manually, then:
+
+### 7.3.1. OpenAMASE Configuration
+
+If your example requires OpenAMASE, you must provide configuration parameters for OpenAMASE in the `amase` section of the YAML file.
+This section is specified as follows:
+
+    amase:
+        scenario: <scenario file>
+        delay: <delay in ms>
+
+The `scenario` parameter must always be specified and the scenario file must exist.
+The scenario file should be an OpenAMASE scenario, as shown above.
+
+The `delay` parameter is optional and is relatively brittle.
+You should instead include the `Test_SimulationTime` service in your OpenUxAS configuration so that your OpenUxAS instances will wait for OpenAMASE to start the simulation before starting their timers.
+The `cfg_WaterwaySearch.xml` includes this service.
+
+### 7.3.2. Single OpenUxAS Configuration
+
+A single instance of OpenUxAS can be specified using the `uxas` section of the YAML file.
+This section is specified as follows:
+
+    uxas:
+        config: <configuration file>
+        bin: <OpenUxAS binary>
+        rundir: <directory for instance outputs>
+
+The `config` parameter must always be specified and the configuration file must exist.
+The configuration file should be an OpenUxAS configuration, as shown above.
+The path to the configuration file is local to the directory containing the YAML file.
+
+The `bin` parameter is optional.
+You should use this parameter if you want to specify that a particular OpenUxAS binary should be used.
+This should be the binary filename, not a path.
+
+The expected use case for this parameter is to support running instances of OpenUxAS that are developed in languages other than C++ (which is the default language for OpenUxAS development).
+For example, if the Ada (partial) implementation of OpenUxAS should be included in an example, the `bin` parameter would be specified to be the binary built by the Ada build process: `uxas-ada`.
+`run-example` will then search for `uxas-ada` on your PATH.
+
+***Note: the `bin` parameter in the YAML file is always followed.***
+That is, if a binary is specified in the YAML file, none of the options provided to the user for controlling which binary is used will be respected.
+Unless there is a clear reason to specify a binary (as in the example above), it is always better to leave the binary unspecified in the YAML file.
+This way, the user has maximum flexibility in selecting the binary they wish for use in the example.
+
+If you do not use this parameter, `run-example` will attempt to find the OpenUxAS binary as follows:
+
+1. based on the command-line option `--uxas-bin` provided to `run-example`;
+2. by looking for `OpenUxAS/obj/cpp/uxas`, which is the output of running `make all`;
+3. by looking for `uxas` on your PATH.
+
+The `rundir` parameter is also optional.
+This is the directory in which output and logs written by OpenUxAS will be placed.
```

Comment:
Might be better to say something along the lines of "run-example will create the stated directory relative to the config.YAML file and will then place files created during runtime (such as the datebase file containing all messages created during execution) in that location

Reply:
I took a stab at this, see if you like my formulation. I left the statement about the path / directory in which the rundir is created for the following sentence, since I think that's a bit clearer.